### PR TITLE
Fix parse action

### PR DESCRIPTION
**Pull request by MarcCote**
When parsing action from tape, data is read using the following convention (block_id, x, y, z), but render.py/world.py expects this convention (x,y,z, block_id).

### CODE REVIEW

File: gridworld/data/adapter/adapter.py [hunk 1]
```diff
@@ -78,28 +61,28 @@ def save_session(self, session, save_path=None):
...
 
         Args:
-            session (GameSession): 
+            session (GameSession):
             visualize (bool, optional): whether to render a high-resolution human friendly visualization
             debug (bool, optional): whether to add a debug info on each screen. Defaults to True.
             postprocess (bool, optional): whether to re-encode video using h264 codec. !!! REQUIRES THE LATEST FFMPEG !!!
         """
-        path = pathlib.Path(outpath or DATA_PREFIX)
+        path = pathlib.Path(outpath or self.dataset.get_data_path())
```
Comment (artemZholus) [inline gridworld/data/adapter/adapter.py:85]: I need to fix this bc of the recently updated `get_data_path` method

Comment (MarcCote) [reply gridworld/data/adapter/adapter.py:85]: Right this PR was based on https://github.com/iglu-contest/gridworld/pull/20

Comment (MarcCote) [reply gridworld/data/adapter/adapter.py:85]: I'll drop 814da99 so it's easier to merge.

**Comment by MarcCote** [comment]
It should be easier to see the proposed fix (ignore whitespace changes).

**Pull request merged**